buildkite/src/Lib/Cmds.dhall

```diff
@@ -67,9 +70,9 @@ let module = \(environment : List Text) ->
     \(cachePath : Text) ->
     \(cmd : CompoundCmd.Type) ->
       let script =
-        "( ${format cmd.postprocess} || true ) && " ++
+        "( ${format cmd.before} || true ) && " ++
```

Comment:
I think what's throwing me off here is that the script is optimistically attempting to perform a command (i.e. untar an archive in this case) on a cache miss.

I somewhat get the idea of logic that maybe the archive already exists so give it a try ... but then the script will go ahead and *produce the data and then tar* unnecessarily it seems if the optimistic check pans out.